Source/FlowEditor/Public/Asset/FlowDebuggerSubsystem.h

```diff
@@ -6,9 +6,77 @@
 #include "Logging/TokenizedMessage.h"
 #include "FlowDebuggerSubsystem.generated.h"
 
+class UFlowGraphNode;
+class UEdGraphPin;
+struct FFlowTraitSettings;
 class UFlowAsset;
 class FFlowMessageLog;
 
+UENUM()
+enum class EFlowTraitType
+{
+	Breakpoint, // default trait type
+
+	// ^ Add new trait types above here ^
+	Max
+};
+
+ENUM_RANGE_BY_COUNT(EFlowTraitType, EFlowTraitType::Max)
+
+// It can represent any trait added on the specific node instance, i.e. breakpoint
+USTRUCT()
+struct FLOWEDITOR_API FFlowDebugTrait
```

Comment:
Renamed FFlowPinTrait to FFlowDebugTrait to better show purpose of this struct. Can rename back if needed)